### ISSUE TITLE
[FEATURE] Open URL that is spelt out

### ISSUE THREAD

**Issue by scorpfromhell** [mozilla/firefox_voice_community#244]
I tried to open the URL 1ct.es by saying "one see tee dot ee es" and firefox opened a new tab with google search results for "1ct es" where the dot is missing.

Since it is a corporate intranet URL, I am forced to either spell out the URL or I would need to add a bookmark and say the name I save that bookmark with.

Bookmarks might be helpful for most frequented sites. But for rarer sites, it might be helpful to be able to spell the URL.

**Comment by jtaylor113** [comment]
I agree with this feature. I have some URLs that I use every day that Firefox voice does not understand. Another option could be to manually go to a URL the first time, then give the current page a "nickname"? I don't think they allow this yet, but would that help when Firefox cannot determine the correct URL?
